internal/file/tar.go

```diff
@@ -16,6 +17,29 @@ const (
 	GB
 )
 
+type errZipSlipDetected struct {
+	Prefix   string
+	JoinArgs []string
+}
+
+func (e *errZipSlipDetected) Error() string {
+	return fmt.Sprintf("potential zip slip attack: prefix=%q dest=%q", e.Prefix, e.JoinArgs)
```

Comment:
Can this be changed to say what isn't allowed instead? I understand that this addresses the zip slip attack, but it assumes I grasp exactly what that is and how to fix it. For example:
```suggestion
	return fmt.Sprintf("paths are not allowed to resolve outside their prefix (%q). Destination: %q", e.Prefix, e.JoinArgs)
```

Probably butchering the message but anything that clarifies here would be great